### ISSUE TITLE
Empty state icon and primary button updates

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-react#2929]
`.pf-c-empty-state__icon` can now be a `div` that allows for an icon, in case that icon is a component, like the `Spinner` component.

Also added a new element `.pf-c-empty-state__primary` that can be used in lieu of `<Button variant="primary">` as the primary button in the empty state component.

Core PR - https://github.com/patternfly/patternfly-next/pull/2254/

**Comment by boaz0** [comment]
can I work on this?

**Comment by rachael-phillips** [comment]
Hi @boaz0 it's not ready yet from a core perspective. Are you blocked or waiting on this?

**Comment by boaz0** [comment]
the latter

**Comment by mcoker** [comment]
@rachael-phillips this work was completed in core as part of https://github.com/patternfly/patternfly-next/pull/2254. This work is needed before react can finish building the empty state and loading state table demos in https://github.com/patternfly/patternfly-react/issues/1433

**Comment by rachael-phillips** [comment]
We are going to pull this up into the prioritized backlog. The prioritized backlog is now the place where we are going to surface issues we want to make visible and top of mind: https://github.com/patternfly/patternfly-react/milestone/20 

We are also going to place this on on hold until the Core work is done.
